pages/index.tsx

```diff
@@ -1,15 +1,31 @@
-import { useEffect, useState } from 'react';
-
 import Trending from '../components/trending/Trending';
 import request from '../modules/request';
 import { AppWrapper, GlobalStyle } from '../styles/index.styles';
 
-const Home = (): JSX.Element => {
-  const [data, setData] = useState([]);
+interface Community {
+  title: string;
+  url: string;
+  members: number;
+  image: string;
+}
+
+interface HomeProps {
+  communities: Community[];
+}
+const Home = ({ communities }: HomeProps): JSX.Element => {
+  return (
+    <AppWrapper>
+      <GlobalStyle />
+      <Trending communities={communities} />
+    </AppWrapper>
+  );
+};
+
+// This function gets called at build time
+export const getStaticProps = async (): Promise<unknown> => {
+  // Call an external API endpoint to get communities
 
-  useEffect(() => {
-    const queryData = async () => {
-      const query = `{
+  const query = `{
```

Comment:
Prettier wont format this multiline string, because its a string. Can we plz format it better? After each `{` the next line should start 2 spaces further inside.

Reply:
Done!